### ISSUE TITLE
Two year reminders for editors

### ISSUE THREAD

**Issue by maelle** [ropensci/software-review-meta#96]
Via https://github.com/ropensci/dev_guide/issues/546#issuecomment-1323718050

**Comment by maelle** [comment]
@mpadge @noamross I had forgotten about this but it'd be nice.

**Comment by mpadge** [comment]
Indeed! Likely best implemented as Airtable -> Slack ping, which'd be the realms of @noamross, right Noam?

**Comment by noamross** [comment]
While I can take some of these we should not divide tasks by platform, you should know the Airtable workflows, too! You have full access @mpadge, but we can pair-develop this in the next couple of weeks if you want.  I think the main thing holding this up has been collecting and recording start dates from all of the editors here: https://airtable.com/app8dssb6a7PG6Vwj/tblIBMYXKsF4DwhEW/viwtCcPZXpi5V8xTX?blocks=hide
